javascript/widgets/authui.js

```diff
@@ -680,7 +680,7 @@ firebaseui.auth.AuthUI.prototype.initializeForAutoUpgrade_ = function(cb) {
  *     null otherwise.
  * @private
  */
-firebaseui.auth.AuthUI.prototype.getUpgradableUser_ = function() {
+firebaseui.auth.AuthUI.prototype.getUpgradeableUser_ = function() {
```

Comment:
It's possible this is a public api, in which case I can drop this change. I assumed it wasn't given the `_` tail.